src/editors/containers/ProblemEditor/data/mockData/olxTestData.js

```diff
@@ -82,11 +82,11 @@ export const checkboxesOLXWithFeedbackAndHintsOLX = {
       },
     ],
   },
-  question: '<p>You can use this template as a guide to the simple editor markdown and OLX markup to use for checkboxes with hints and feedback problems. Edit this component to replace this template with your own assessment.</p><bold>Add the question text, or prompt, here. This text is required.</bold><em>You can add an optional tip or note related to the prompt like this.</em>',
+  question: '<p>You can use this template as a guide to the simple editor markdown and OLX markup to use for checkboxes with hints and feedback problems. Edit this component to replace this template with your own assessment.</p><strong>Add the question text, or prompt, here. This text is required.</strong><em>You can add an optional tip or note related to the prompt like this.</em>',
```

Comment:
I just updated the tag so that it actually applied the formatting it was trying to apply. The legacy experience has `<label>` and `<description>` which returned a different style than the new experience. So I think that  we are cool with this change.

Reply:
ah makes sense. This change is to the html which is what question is translated into, not olx. gotcha.